packages/frontend/app/components/visualizer-program-year-objectives.js

```diff
@@ -16,15 +17,29 @@ export default class VisualizerProgramYearObjectivesComponent extends Component
   @tracked programYearName;
   @tracked data;
 
-  load = restartableTask(async (element, [programYear]) => {
-    const cohort = await programYear.cohort;
-    const year = await programYear.getClassOfYear();
+  constructor() {
+    super(...arguments);
+    this.load.perform();
+  }
+
+  load = restartableTask(async () => {
+    const cohort = await this.programYear.cohort;
+    const year = await this.programYear.getClassOfYear();
     const classOfYear = this.intl.t('general.classOf', { year });
     this.programYearName = cohort.title ?? classOfYear;
 
-    this.data = await this.getData(programYear);
+    this.data = await this.getData(this.programYear);
```

Comment:
This doesn't time out correctly if the program year isn't already loaded. It *could* return null.  This fix is a bit more complicated, but worth the time:
1. Remove the `load` entirely
2. Create a `TrackedAsyncData` that calls `getData` using the program year. Until program year exists it should return null.
3. Probably lots of adjustment and cleanup to get that to work

Reply:
@jrjohnson Any idea why the links to these visualizations only exist on local, but not dev?

Reply:
@michaelchadwick the visibility of these links is controlled via a feature flag. 

for reference, see:
https://github.com/ilios/frontend/blob/master/packages/frontend/app/components/program-year/overview.hbs#L7